codegen/Package.swift

```diff
@@ -99,11 +99,11 @@ if let smithySwiftDir = ProcessInfo.processInfo.environment["SMITHY_SWIFT_CI_DIR
    let sdkDir = ProcessInfo.processInfo.environment["AWS_SDK_SWIFT_CI_DIR"] {
     package.dependencies += [
         .package(name: "ClientRuntime", path: smithySwiftDir),
-        .package(name: "AWSClientRuntime", path: "\(sdkDir)/AWSClientRuntime"),
+        .package(name: "AWSSwiftSDK", path: sdkDir),
```

Comment:
same as above.

Reply:
See my comment above, this package dependency is correct.